src/Random.mo

```diff
@@ -77,7 +77,10 @@ module {
       var pp = p;
       var acc : Nat = 0;
       for (i in it) {
-        if (8 : Nat8 <= pp) { acc := acc * 256 + Prim.nat8ToNat(i) } else if (0 : Nat8 == pp) {
+        if (8 : Nat8 <= pp) {
```

Comment:
this looks like reformatting

Reply:
Yeah, this was prettier reformatting your code to make it less legible ;->

I reformatted it to something I could read.

I think prettier did that because your then branch was just a single line
 
```
if (8 : Nat8 <= pp)
 { acc := acc * 256 + Prim.nat8ToNat(i) } 
 else ...
```
I'm hoping
```
if (8 : Nat8 <= pp) { 
  acc := acc * 256 + Prim.nat8ToNat(i)
} else ...
```
gets preserved.